src/main/java/opendota/Parse.java

```diff
@@ -146,7 +154,20 @@ public UnknownItemFoundException(String message) {
 	private GreevilsGreedVisitor greevilsGreedVisitor;
 	private TrackVisitor trackVisitor;
     private ArrayList<Boolean> isPlayerStartingItemsWritten;
-    
+
+    //Draft stage variable
+    Integer draftStage = 0;
+    int[] draftHeroes = new int[20];
+    int[] draftOrder = new int[20];
+    int[] draftActiveTeam =  new int[20];
+    int[] draftTime = new int[20];
+    double[] draftExTime0 = new double[20];
+    double[] draftExTime1 = new double[20];
+
+    int order = 1;
```

Comment:
why does this start at 1 and not 0?

Reply:
I guess the data from valve is 1-indexed so we should start at 1 for consistency

Reply:
Indexing by 1 is my default heh